cosmos/gaia
# feat(e2e): add e2e tests for the slashing module

close #1679

## Description

Add tests for the slashing module.

### Test workflow

- Create a jailed keys and the validator into the genesis JSON;
- Iterate over all validators and find the jailed one;
- Execute the Unjail gaiad command for the jailed validator;
- Query the validator and check if is already unjailed;
